app/views/includes/motif-tagging.pug

```diff
@@ -4,13 +4,13 @@ h2 Emneord
 
 .row.motif-tagging__editing-container
   - const tags = helpers.motifTagging.getVisionTags(metadata) || [];
-  if tags && tags.length
```

Comment:
A negated negative field is a loaded shotgun pointet downwards :D

I'd prefer the flag to be called config.features.enableVisionTagSuggestions